[31mCRITICAL[0m Test_Orange_HRM:test_OrangeHRM_Login.py:23 test_Orange_Hrm_Login_001 is started
[33mWARNING [0m urllib3.connectionpool:connectionpool.py:870 Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000209D70DD010>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/dfe60def0de287453c72e7316abc3348
[33mWARNING [0m urllib3.connectionpool:connectionpool.py:870 Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000209D70DDF70>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/dfe60def0de287453c72e7316abc3348
[33mWARNING [0m urllib3.connectionpool:connectionpool.py:870 Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000209D70DE240>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/dfe60def0de287453c72e7316abc3348